apisix/plugins/log-rotate.lua

```diff
@@ -117,6 +121,22 @@ local function rotate_file(date_str, file_type)
     local ok, msg = os.rename(file_path_org, file_path)
     core.log.info("move file from ", file_path_org, " to ", file_path,
                   " res:", ok, " msg:", msg)
+
+    if ok and enable_compression then
+        local compression_filename = new_filename .. COMPRESSION_FILE_SUFFIX
+        local cmd = string.format("cd %s && tar -zcf %s %s",
```

Comment:
Need to clarify in the doc that this feature requires `tar` installed.

Reply:
okay, thanks, done

Reply:
Can we test whether "tar" is installed? If not installed, give a tip and then set `enable_compression = false`. (Just a question, if it is wrong, pls ignore this.)